### ISSUE TITLE
cargo_bpf example not compiling

### ISSUE THREAD

**Issue by rtkaratekid** [redsift/redbpf#39]
Exactly following the instructions on the [cargo_bpf](https://redsift.github.io/rust/redbpf/doc/cargo_bpf/) page leads to compiler errors for "unresolved imports." Full error message below:

```sh
error[E0432]: unresolved import `redbpf_probes`
 --> src/block_http/main.rs:7:1
  |
7 | program!(0xFFFFFFFE, "GPL");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | |
  | no `bpf_trace_printk` in `helpers`
  | in this macro invocation

error: cannot determine resolution for the macro `ufmt::uwrite`
 --> src/block_http/main.rs:7:1
  |
7 | program!(0xFFFFFFFE, "GPL");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ in this macro invocation
  |
  = note: import resolution is stuck, try simplifying macro imports

warning: due to multiple output types requested, the explicitly specified output file name will be adapted for each output type

warning: ignoring --out-dir flag due to -o flag

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.
error: could not compile `hello-bpf`.

To learn more, run the command again with --verbose.
error: failed to compile the `block_http' program
```

Similarly, the examples given in [this blog post](https://blog.redsift.com/labs/writing-bpf-code-in-rust/) throws the exact same errors.
I'd be happy to correct the issue, but I'm still a newcomer to Rust and might not be able to make the fix as quickly as I would like.

Running Ubuntu 18.04/Linux 4.15 - BCC and bpftrace work as intended. Compiling C code into BPF byte code with clang also works as intended.

**Comment by rsdy** [comment]
Ok, so I went through this. Thanks for reporting this, it actually looks like there were two issues here:

  1. Somehow publishing of v0.9.10 of redbpf-probes failed. This was a mistake. Please `cargo update`
  2. You'll need to `use redbpf_probes::net::NetworkBuffer;` in `main.rs`, because the latest redbpf-probes requires this.

After these, you should be able to build the example code.

**Comment by rtkaratekid** [comment]
So I ran ```cargo update``` and then I wasn't sure if I was supposed to replace any of the other use clauses, so all I did was add ```use redbpf_probes::net::NetworkBuffer;``` and it worked like a charm. 
I'm looking forward to poking around in the code more and building things/contributing! Ingraind looks cool, but for now I'm more interested in a pure BPF-frontend. If I come across anything else or suggestions I'll let you know.
Thanks for the help!
